packages/utils/src/socialRemark/README.md

```diff
@@ -0,0 +1,84 @@
+# SocialRemark
+
+SocialRemark protocol is designed to process cross-chain actions.
+
+## SocialRemark message format
+
+- Domain Registration
+  `prot_name::version::action::op_id::target::domain_name::token`
+- Energy Generation
+  `prot_name::version::action::op_id::target::energy_amount::token`
+
+Remark message has 2 parts:
+- core part (is the same for all messages): `prot_name::version::action`
+- content part (depends on actions):
+  - `op_id::target::domain_name::token`
+  - `op_id::target::energy_amount::token`
+
+Core part options:
+- `prot_name` - protocol name (default valid value `social` but can be configured but `.setConfig()` method)
+- `version` - protocol version (current available version `0.1`)
+- `action` - message action (available actions: `DMN_REG`, `DMN_REG_OK`, `DMN_REG_REFUND`, `NRG_GEN`, `NRG_GEN_OK`, `NRG_GEN_REFUND`)
+
+Content options:
+- `op_id` - unique operation identified. Should be the same for all actions of one operation (the same `op_id` value for `DMN_REG` , `DMN_REG_OK` , `DMN_REG_REFUND` of A single domain registration flow. Necessary for relating actions with each other.)
+- `target` - recipient of domain or energy _(address in any SS58 format, SocialRemark instance will convert it to SubSocial format underhood (prefix: 28))_
```

Comment:
```suggestion
- `target` - The recipient of a domain or energy _(this can be an address in any SS58 format, the SocialRemark instance will convert it to Subsocial's format under the hood (prefix: 28))_
```
"recipient of a domain" implies that a domain is being transferred. Is that the only time you would use `target`?

Reply:
@Yung-Beef

Reply:
```suggestion
- `target` - The receiver of a domain or recipient of energy _(this can be an address in any SS58 format, the SocialRemark instance will convert it to Subsocial's format under the hood (prefix: 28))_
```
Or should it be "the owner of a new domain"?

Reply:
`receiver`  is fine I guess